packages/styles/src/components/button.scss

```diff
@@ -105,8 +108,15 @@ tokens.$default-map: components.$post-button;
   }
 }
 
-// Tertiary with no padding (overrides the padding defined by the sizing classes above)
 .btn-link {
-  padding-inline-start: 0;
-  padding-inline-end: 0;
+  text-decoration: tokens.get('link-decoration', elements.$post-link);
+  border-radius: tokens.get('link-border-radius', elements.$post-link);
+  min-height: 0;
+  font-size: inherit;
+  font-weight: inherit;
+  padding: 0;
```

Comment:
```suggestion
  display: inline;
  text-decoration: tokens.get('link-decoration', elements.$post-link);
  border: 0 none;
  border-radius: tokens.get('link-border-radius', elements.$post-link);
  min-height: 0;
  font-size: inherit;
  font-weight: inherit;
  padding: 0;
```
Even with this changes, the button has still a different height than an anchor element:
![image](https://github.com/user-attachments/assets/1b4ef949-7c52-4236-85a9-d9270762742e)

To fix this, we could add `display: inline-block` to the anchor styles, but I'm not sure if this would produce unwanted side-effects.
![image](https://github.com/user-attachments/assets/c2633b3c-3c70-488f-94c3-c581711cfb46)

So I'd say, we go with the solution suggested above.

Reply:
I commited your suggestion but then it doesn't look good if there is an icon, so I just removed the display: inline and it seems to work better that way.
![Screenshot 2024-12-12 100905](https://github.com/user-attachments/assets/7e93972d-8c32-4dfb-b611-582400a877f3)
![Screenshot 2024-12-12 100939](https://github.com/user-attachments/assets/77bbb676-4c94-4c7d-8630-547e2f3f9a46)